Makefile

```diff
@@ -140,7 +140,14 @@ cert_manager: remove_cert_manager
 remove_cert_manager:
 	- $(CLUSTER_CLIENT) delete -f $(CERT_MANAGER_MANIFEST)
 
-
+.PHONY: check_cert_manager
+check_cert_manager:
+       @api_versions=$$(kubectl api-versions) &&\
+       if [ $$(echo "$${api_versions}" | grep -c '^cert-manager.io/v1$$') -eq 0 ]; then if [ "$${DISABLE_SERVICE_TLS}" != "true" ]; then\
+                       echo 'cert-manager is not installed, install using "make cert_manager" or disable TLS for services by setting DISABLE_SERVICE_TLS to true' >&2\
+                       && exit 1;\
+               fi;\
+       fi
```

Comment:
These lines seem to be indented with spaces instead of tabs, because of this Make doesn't recognize them as a recipe.

Reply:
My bad, I was sure it worked when I tested it out but it must have missed it.